src/Chapter18/18.11.SpecifyingAReturnAttribute.DescriptionAttribute.cs

```diff
@@ -4,8 +4,8 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter18.Listing18_11
 {
     public class DescriptionAttribute : Attribute
     {
-        private string description;
-		public string Description { get { return Description; } }
+        private readonly string description;
+		public string Description { get { return description; } }
```

Comment:
I realize this matches the original, but we should fix the naming of the field.
```suggestion
        private readonly string _Description;
	public string Description { get { return _Description; } }
```